src/Layout.ts

```diff
@@ -1861,11 +1752,12 @@ export class Union extends Layout {
  *
  * @augments {Layout}
  */
-export class VariantLayout extends Layout {
+export class VariantLayout extends Layout<LayoutObject> {
+  property: string;
```

Comment:
Added stricter `property` types for subclasses where they are known to be defined